docs/docs/glossary.md

```diff
@@ -0,0 +1,19 @@
+# Glossary
+
+- [Glossary](#glossary)
+  - [AVS (Actively Validated Service)](#avs-actively-validated-service)
+  - [AVS Node](#avs-node)
+  - [Strategy contract](#strategy-contract)
+
+
```

Comment:
Can we add an entity for Eigenlayer Operator?

Eigenlayer Operators act as AVS nodes by opting in to validating AVSs by running their AVS software.

Reply:
Done [c9ed26e](https://github.com/NethermindEth/el-node-spec/pull/58/commits/c9ed26e9d2d3a7d95fb99b8dc0678e22cd688b5e)

Reply:
Thanks. Can't resolve comments, so feel free to resolve.